pkg/config/initoptions.go

```diff
@@ -140,21 +147,21 @@ func isDirectory(path string) bool {
 // config files. Assumes all namespaced resources are in the
 // same namespace. Returns the default namespace if none of the
 // config files has a namespace.
-func calcPackageNamespace(packageDir string) (string, error) {
+func calcPackageNamespace(packageDir, defaultNamespace string) (string, error) {
```

Comment:
We probably want to add this new parameter (and what it means) to the function documentation.